app/requests_handlers.py

```diff
@@ -1281,16 +1280,15 @@ async def get(self, request):
         self.setup_request(request)
 
         display_region = 'ni'
-        page_title = 'You need to visit the Communal Establishment Portal'
+        page_title = 'You need to visit the Communal Establishment Manager Portal'
         locale = 'en'
 
         self.log_entry(request, display_region + '/requests/paper-questionnaire/ce-manager')
 
         return {
                 'page_title': page_title,
                 'display_region': display_region,
```

Comment:
display_region not used in template and could be removed.

Reply:
I'll do the same as above